Example of a station layout [SZCZ]

pecenvilem opened the pull request:
Dear all,
here I submit an example of an OP (Station "Kladno-Dubí", slightly modified from the original) from the network of Správa železnic [SZCZ - Czechia]. Unfortunately I don't know how to work with RDF files, so I am only giving a picture.

Things I would like to point out about the example:
- There is a group exit signal - how it this modelled using the ontology?
- The home signal is located on the left side of the track, which is expressed by a separate marker signal. Can the marker and the home signal be associated with each other in any way?

I was not able to come up with any particularly interesting example for a SoL...

Thank you for any feedback.
Vilém Pecen

PS: Sorry for not responding sooner, I was on holiday...

Certiman reviewed (changes requested):
Many thanks Vilem. In order to reply with a representation, could you submit a file with higher resolution?

pecenvilem commented:
Hello, @Certiman, I have changed the picture and added some comments. For reference, [this video](https://youtu.be/HWA5IXSciOk?si=UCjEHI8wOxkk_Vr0&t=18) shows the location in real life.
